modules/platforms/cpp/odbc-test/include/test_utils.h

```diff
@@ -57,6 +57,27 @@
         return; \
     }
 
+/**
+ * Copy std::string to buffer.
+ *
+ * @param dst Destination buffer.
+ * @param src Source std::string.
+ * @param n Copy at most n bytes of src.
+ */
+inline void CopyStringToBuffer(char *dst, const std::string& src, size_t n) {
+    if (n == 0) {
+        return;
+    }
+
+    size_t size = std::min(src.size(), n - 1);
+
+    if (size != n) {
```

Comment:
This check is not needed actually. It can never be `n`, as it at most `n - 1`

Reply:
Yep, sorry